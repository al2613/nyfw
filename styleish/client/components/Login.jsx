import React from 'react';
import { TitleWrapper, AboutText, Button } from '../styles/Title.jsx';

class Login extends React.Component {
  render() {
    return (
      <TitleWrapper>
        <h1>Login</h1>
        <form>
            Username: <input type= "text"></input>
            <br></br>
            <br></br>
            Password: <input type = "text"></input>
        </form>
      </TitleWrapper>
    )
  }
}
export default Login