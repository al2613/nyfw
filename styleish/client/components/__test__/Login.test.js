import React from 'react';
import { create } from 'react-test-renderer';
import Login from '../Login.jsx';

test('Login snapshot', () => {
  const login = create(<Login />);
  expect(login.toJSON().toMatchSnapshot());
});
