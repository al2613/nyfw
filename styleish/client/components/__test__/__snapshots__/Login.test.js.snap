// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login /> matches snapshot 1`] = `
<div
  className="sc-bdVaJa efbDat"
>
  <h1>
    Login
  </h1>
  <form
    action="http://localhost:3000/login"
  >
    Username: 
    <input
      type="text"
    />
    <br />
    <br />
    Password: 
    <input
      type="text"
    />
  </form>
</div>
`;
